.travis.yml

```diff
@@ -19,21 +17,34 @@ addons:
       - python-coverage
       - python-werkzeug
       - python-psycopg2
-      - postgis
-      - postgresql-9.4-postgis-2.1
+      - libgdal1-dev
```

Comment:
@alculquicondor almost the problem was that libgdal1-dev would unintsall postgres installation because they were conflicting so both were installed but postgres was removed afterwards. I think you might be saying the same thing.